### ISSUE TITLE
How to set the event trigger for Crisp on the Google Tag Manager?

### ISSUE THREAD

**Issue by ghost** [crisp-im/crisp-snippets#1]
Thanks for your great work @mywaystar 

I have some question in using this code snippet.

So, I've added the custom variables to the google tag manager. For example,
eventCategory, eventAction,  eventLabel. 

Btw, I am a little confused in using `hitType`.
Do I need to use the `hitType` label as an eventName in trigger configuration? Or do I need to set the `event` as an eventName?
I am sharing a screenshot below to clear the current problem.

![image](https://user-images.githubusercontent.com/24935417/36127058-ba1f252e-105b-11e8-8a21-00d23ea15426.png)

```
$crisp.push(["on", "chat:opened", function(){
	ga('send', {
	  hitType: 'event',      // I am just confused about this setting.
	  eventCategory: 'Crisp',
	  eventAction: 'chat:opened',
	  eventLabel: 'Chat Opened'
	});
}]);
```

In short words, I'd like to know about the event trigger setting for Crisp on the Google Tag Manager.

I look forward to hearing back the good news from you.

Thanks in advance.
